### DISCUSSION TITLE
Ironman trivialised by the round ending

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14042] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The point of ironman mode is that if your character dies, then you're out of the game. One might imagine that if the entire crew dies, then that's just the end of the campaign. This is not the case. If everyone dies, the server goes back to lobby like normal, and then you can simply resume the save at the start of the level. All stakes have been lost, since even if the entire crew dies, you can just try again, which really goes against the spirit of an ironman feature.

### Reproduction steps

1. start an ironman campaign
2. get the entire crew to jump out the airlock
3. wait 60 seconds (or just press the "server lobby" button
4. continue your "ironman" save after everyone just died, and you will find that everyone is now alive and you can try again

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
To me this does not seem like how the iron man mode should work either, I would've expected the campaign to end when everyone is dead too. Creating a ticket.

**Comment by Jasontti** [comment]
Tested the fix.
Character data is now saved at death.
Closing discussion as resolved.
